### PR TITLE
Several changes to remove NP-variables

### PR DESCRIPTION
**Pull request by waja**
Feel free to catch up part of this.

### CODE REVIEW

**Comment by sni** [comment]
We finally should do this. Its not easily possible anymore to share patches between NP and MP anyway.

**Comment by waja** [comment]
>  Its not easily possible anymore to share patches between NP and MP

I wouldn't say that in general. We exchanged still patches in the recent past.
Anyway, this PR needs needs to be reworked in this case. Maybe recreating might be easier, dunno.

File: configure.ac [hunk 1]
```diff
@@ -1330,17 +1330,17 @@ fi
 
 AC_MSG_CHECKING([for number of online cpus])
 AC_TRY_COMPILE([#include <unistd.h>],
-	[sysconf(_SC_NPROCESSORS_ONLN) > 0;],
-	AC_DEFINE(HAVE_SYSCONF__SC_NPROCESSORS_ONLN,1,[Define if sysconf returns number of online cpus])
-	AC_MSG_RESULT([sysconf(_SC_NPROCESSORS_ONLN)]),
+	[sysconf(_SC_MPROCESSORS_ONLN) > 0;],
```
Comment (glensc) [inline configure.ac]: haha, dumb replace: _SC_MPROCESSORS_ONLN

**Comment by RincewindsHat** [comment]
Just tried to get this somewhere close to master, but got lots of merge conflicts. This has to be done from start.

**Pull request closed without merging**